commit f41d933a784e76a26b1c7e37a73404dc8079b37b
Author: Takashi Iwai <tiwai@suse.de>
Date:   Tue Feb 28 17:16:48 2017 +0100

    ALSA: ctxfi: Fallback DMA mask to 32bit
    
    commit 15c75b09f8d190f89ab4db463b87d411ca349dfe upstream.
    
    Currently ctxfi driver tries to set only the 64bit DMA mask on 64bit
    architectures, and bails out if it fails.  This causes a problem on
    some platforms since the 64bit DMA isn't always guaranteed.  We should
    fall back to the default 32bit DMA when 64bit DMA fails.
    
    Fixes: 6d74b86d3c0f ("ALSA: ctxfi - Allow 64bit DMA")
    Signed-off-by: Takashi Iwai <tiwai@suse.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1235_f41d933a784e76a26b1c7e37a73404dc8079b37b.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1235_f41d933a784e76a26b1c7e37a73404dc8079b37b.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
